packages/ui-components/src/components/DataPreviewTable/DataPreviewTable.js

```diff
@@ -27,7 +27,7 @@ const getColumnId = ({ id, accessor, Header }) => {
   return Header;
 };
 
-export const DataTable = ({ columns, data, className, rowLimit }) => {
+export const DataPreviewTable = ({ columns, data, className, rowLimit }) => {
```

Comment:
Hmm... it kind of is a "DataTable" though. In our case we call it a preview but you're actually running it, it's just that you haven't made it visible to users anywhere.

Reply:
Your new component ParameterList could live under a folder "data-table-editing" or something like that, or maybe under DataTable/editor.

SQLQueryEditor is just a generic component that can live in folder Inputs.

Reply:
Yeah make sense, adjusted.